src/components/ui/upload/PreviewFile.tsx

```diff
@@ -21,19 +22,19 @@ const PreviewFile = ({ file, allowDelete = false, onDelete }: Props) => {
 
   return (
     <div className="relative">
-      <img
+      <Image
         className="empty-box object-cover"
         src={imageUrl}
         alt={t('file.preview')}
       />
       {allowDelete && (
-        <button
+        <Button
           onClick={onDelete}
           aria-label={t('button.delete_file')}
```

Comment:
Laten we in een nieuwe PR een IconButton component maken, zodat we aria-labelledby kunnen gebruiken ipv aria-label

Reply:
[Issue](https://github.com/delta10/signalen-frontend-wcag/issues/77) voor aangemaakt